cmd/kpod/diff.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	diffFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:   "archive",
+			Usage:  "Save the diff as a tar archive",
+			Hidden: true,
+		},
+		cli.BoolFlag{
+			Name:   "json",
+			Usage:  "Format output as JSON",
+			Hidden: true,
+		},
+	}
+	diffDescription = "Inspect changes on a container, layer, or image's filesystem.  The layer (or top layer of container/image) will be compared to its parent layer"
```

Comment:
I have a feeling this does not display well on the kpod diff --help screen.
You can use 
diffDescription = `Inspect changes on a container, layer, or image's filesystem.  
The layer (or top layer of container/image) will be compared to its parent layer
`
To make it look better on the screen

`